inverse_covariance/clean.py

```diff
@@ -0,0 +1,323 @@
+import numpy as np
```

Comment:
I addressed by just changing the file names.  If we want to subscope it to `clean` or `preprocessing` later we can but since this is the only one, keeping it flat for now.

Reply:
*addressed when I push changes